docs/src/pages/[platform]/theming/style-attributes/index.page.mdx

```diff
@@ -0,0 +1,322 @@
+  alignItems="stretch"
+  alignContent="flex-start"
+  gap={tokens.space.medium}
+>
+  <Button>Button 1</Button>
+  <Button>Button 2</Button>
+  <Button>Button 3</Button>
+</Flex>;
+
+//using token names
+<Flex
+  justifyContent="flex-start"
+  alignItems="stretch"
+  alignContent="flex-start"
+  gap="medium"
+>
+  <Button>Button 1</Button>
+  <Button>Button 2</Button>
+  <Button>Button 3</Button>
+</Flex>;
+```
+
+<LayoutStyles />
+
+### Margin and Padding
+
+```tsx
+import { Button, useTheme } from '@aws-amplify/ui-react';
+
+//use any CSS value
+<Button padding="1rem">Padded Button</Button>
+
+//use design tokens
+const { tokens } = useTheme();
+<Button margin={tokens.space.medium}>Button with margin</Button>
+
+//use design token name
+<Button padding="relative.large">Padded Button</Button>
+```
+
+<MarginAndPaddingStyles />
+
+### Position Styles
+
+```tsx
+import { View, useTheme } from '@aws-amplify/ui-react';
+
+//use any CSS value
+<View position="relative" width="200px" height="200px">
+  <View position="absolute" left="10%" top="20%" width="20px" height="20px" />
+</View>;
+
+//using design tokens
+const { tokens } = useTheme();
+<View position="relative" width="200px" height="200px">
+  <View
+    position="absolute"
+    left={tokens.space.relative.xxl}
+    top={tokens.space.relative.xxxl}
+    width={tokens.space.large}
+    height={tokens.space.large}
+  />
+</View>;
+
+//using design token names
+<View position="relative" width="200px" height="200px">
+  <View
+    position="absolute"
+    left="relative.xxl"
+    top="relative.xxxl"
+    width="large"
+    height="large"
+  />
+</View>;
+```
+
+<PositionStyles />
+
+### Size Styles
+
+```tsx
+import { View, useTheme } from '@aws-amplify/ui-react';
+
+//use any css value
+<View height="100vh" width="100vw"> Full View </View>
+
+//use design tokens
+const { tokens } = useTheme();
+<View height={tokens.space.relative.full} width={tokens.space.relative.full}> Full View </View>
+
+//use token names
+<View width="large" height="large" maxHeight="relative.xl" maxWidth="relative.xl"> Large View </View>
+```
+
+<SizeStyles />
+
+### Shadow Styles
+
+```tsx
+import { Button, useTheme } from '@aws-amplify/ui-react';
+
+//use any css value
+<Button boxShadow="10px 5px 5px black">Shadow Button</Button>;
+```
+
+<ShadowStyles />
+
+### Typography Styles
+
+```tsx
+import { Text, useTheme } from '@aws-amplify/ui-react';
+
+//use any css value
+<Text fontSize="24px" textDecoration="underline">Typography text</Text>
+
+//use design tokens
+const { tokens } = useTheme();
+<Text fontSize={tokens.fontSizes.large}>Typography text</Text>
+
+//use token names
+<Text fontSize="xl">Typography text</Text>
+```
+
+<TypographyStyles />
+
+### Other Styles
+
+Below is a list of other style properties accepted by amplify-ui primitives.
+
+<OtherStyles />
+
+### Design Tokens
+
+Below is a table that outlines the available design tokens which can be used with the style properties.
```

Comment:
Can we allow the Design Tokens expander to be collapsible?

Reply:
I think we may want to just link to the Default Theme page that @hbuchel put together here? Otherwise it feels redudant.

Reply:
Design tokens removed and links to the theme page design tokens have been added.